client/src/components/overview/GalleryView.jsx

```diff
@@ -85,12 +82,15 @@ function findAverageRating(ratings)  {
 
 // This is the actual functional component
 function Overview(props) {
+  var [expanded, setView] = useState(false);
```

Comment:
consider using the same variable declaration style. let/const OR var

Reply:
Oh yeah, good call. There were inconsistencies because the way I use the set functions. If I am changing the state, sometimes the const won't let me do it.